app/templates/components/public/side-menu.hbs

```diff
@@ -9,7 +9,7 @@
           {{t 'Tickets'}}
         </a>
       {{/if}}
-      {{#if this.showSpeakers}}
+      {{#if (and this.showSpeakers this.event.speakers.length)}}
```

Comment:
This is not good. Now it'll load speakers on the index page itself. Check the problem in showSpeakers and fix that instead